docroot/profiles/custom/cgov_site/modules/custom/cgov_article/config/install/core.entity_form_display.node.cgov_article.default.yml

```diff
@@ -24,17 +25,17 @@ dependencies:
     - field.field.node.cgov_article.field_short_title
     - field.field.node.cgov_article.field_site_section
     - node.type.cgov_article
-    - workflows.workflow.editorial_workflow
```

Comment:
Need to look into why this was removed. Likely not an issue but will follow up.

Reply:
There was a fix in 8.7 to remove the moderation state from content types that did not participate in moderation. After that fix it started disappearing/reappearing. We attach the content types to workflow in their .install file.  Otherwise, the workflow config has all the content types as dependencies.